http-client-core/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation libs.managed.reactor
 
     api project(":runtime")
+    api project(":management")
```

Comment:
this management dependency can't break required here

Reply:
@graemerocher When you day management cannot be a dep. do you want me to move that indicator under management then? From my understanding to keep it under http-client-core it would need that dep as either api or implementation?

Reply:
needs to be `compileOnly`

Reply:
or moved to management module